scripts/keychain-cli.js

```diff
@@ -935,6 +938,25 @@ program
         }
     });
 
+program
+    .command('encrypt-wallet <pass-phrase> [new-phrase]')
+    .description('Encrypt wallet or update passphrase')
+    .action(async (passphrase, newPassphrase) => {
```

Comment:
Should the CLI set the env var? I expected to be able to use the CLI after encrypting but instead I get
`Wallet encrypted but KC_ENCRYPTED_PASSPHRASE not set`

Reply:
Good idea. I'll add this.